pepys_import/file/file_processor.py

```diff
@@ -319,12 +306,51 @@ def process_file(self, file_object, current_path, data_store, processed_ctr, imp
 
             # If all tests pass for all parsers, commit datafile
             if not errors:
-                log = datafile.commit(data_store, change.change_id)
-
                 # Keep track of some details for the import summary
                 summary_details = {}
                 summary_details["filename"] = basename
 
+                log = datafile.commit(data_store, change.change_id)
+                metadata_summaries_after = data_store.get_status(
+                    report_metadata=True, exclude=exclude
+                )
+                measurement_summaries_after = data_store.get_status(
+                    report_measurement=True, exclude=exclude
+                )
+                print(
+                    metadata_summaries_after.show_delta_of_rows_added_metadata(
+                        metadata_summaries_before, title="METADATA REPORT"
```

Comment:
Do we want the titles to be in ALL CAPS? This might be something that @clenoble has an opinion on.